test/Table_Tests/src/Common_Table_Operations/Nothing_Spec.enso

```diff
@@ -0,0 +1,253 @@
+                table.at "x" . is_in (table.at "y") . to_vector . should_equal [Nothing, Nothing]
+
+            Test.specify "Correctly handle Nothing in: Nothing.is_in Column with Nothings (returning Nothing)" <|
+                table.at "n" . is_in (table.at "x") . to_vector . should_equal [Nothing, Nothing]
+
+            Test.specify "Correctly handle Nothing in: "+value_type.to_text+".is_in Vector (returning True)" <|
+                table.at "x" . is_in (table.at "z" . to_vector) . to_vector . should_equal [true_if_not_nothing, Nothing]
+
+            Test.specify "Correctly handle Nothing in: "+value_type.to_text+".is_in Vector with Nothings (returning True)" <|
+                table.at "x" . is_in (table.at "x" . to_vector) . to_vector . should_equal [true_if_not_nothing, Nothing]
+
+            Test.specify "Correctly handle Nothing in: "+value_type.to_text+".is_in Vector with Nothings (returning Nothing)" <|
+                table.at "x" . is_in (table.at "y" . to_vector) . to_vector . should_equal [Nothing, Nothing]
+
+            Test.specify "Correctly handle Nothing in: Nothing.is_in Vector with Nothings (returning Nothing)" <|
+                table.at "n" . is_in (table.at "x" . to_vector) . to_vector . should_equal [Nothing, Nothing]
+
+    Test.group prefix+"Boolean is_in" <|
+        make_containing_values had_null had_true had_false =
+            null_maybe = if had_null then [Nothing] else []
+            true_maybe = if had_true then [True] else []
+            false_maybe = if had_false then [False] else []
+            null_maybe + true_maybe + false_maybe
+
+        # [input, had_null, had_true, had_false, output]
+        cases = [[True, True, True, True, True]]
+            + [[True, True, True, False, True]]
+            + [[True, False, True, True, True]]
+            + [[True, False, True, False, True]]
+            + [[True, True, False, True, Nothing]]
+            + [[True, True, False, False, Nothing]]
+            + [[True, False, False, True, False]]
+            + [[True, False, False, False, False]]
+            + [[False, True, True, True, True]]
+            + [[False, True, False, True, True]]
+            + [[False, False, True, True, True]]
+            + [[False, False, False, True, True]]
+            + [[False, True, True, False, Nothing]]
+            + [[False, True, False, False, Nothing]]
+            + [[False, False, True, False, False]]
+            + [[False, False, False, False, False]]
+            + [[Nothing, True, True, True, Nothing]]
+            + [[Nothing, True, False, True, Nothing]]
+            + [[Nothing, False, True, True, Nothing]]
+            + [[Nothing, False, False, True, Nothing]]
+            + [[Nothing, True, True, False, Nothing]]
+            + [[Nothing, True, False, False, Nothing]]
+            + [[Nothing, False, True, False, Nothing]]
+            + [[Nothing, False, False, False, False]]
+
```

Comment:
IMO we should generate this programmatically, it is very hard to parse and check this huge table for correctness.

Reply:
Done.